.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const Module = require('module');
+const path = require('path');
+
+const NodePlugin = require('./tools/node_modules/eslint-plugin-node-core');
+NodePlugin.RULES_DIR = path.resolve(__dirname, 'tools', 'eslint-rules');
+
+const ModuleFindPath = Module._findPath;
+const hacks = ['eslint-plugin-node-core', 'eslint-plugin-markdown', 'babel-eslint'];
+Module._findPath = (request, paths, isMain) => {
+  const r = ModuleFindPath(request, paths, isMain);
+  if (!r && hacks.includes(request)) {
+    try {
+      return require.resolve(`./tools/node_modules/${request}`);
+    } catch (err) {
+      return require.resolve(
+        `./tools/node_modules/eslint/node_modules/${request}`);
+    }
+  }
+  return r;
+};
+
+module.exports = {
+  root: true,
+  plugins: ['markdown', 'node-core'],
+  env: { node: true, es6: true },
+  parser: 'babel-eslint',
+  parserOptions: { sourceType: 'script' },
+  overrides: [
+    {
+      files: [
+        'doc/api/esm.md',
+        '*.mjs',
+        'test/es-module/test-esm-example-loader.js',
+      ],
+      parserOptions: { sourceType: 'module' },
+    },
+  ],
+  rules: {
```

Comment:
Can you either add back the sections from the old format or (my preference) sort this object alphabetically?